sdks/python/apache_beam/dataframe/doctests.py

```diff
@@ -290,34 +298,60 @@ def to_callable(cond):
     super(BeamDataframeDoctestRunner, self).__init__(
         checker=_DeferrredDataframeOutputChecker(self._test_env, use_beam),
         **kwargs)
+    self.skipped = 0
+    self.wont_implement = 0
 
   def run(self, test, **kwargs):
     self._checker.reset()
-    if test.name in self._skip:
-      for example in test.examples:
-        if any(should_skip(example) for should_skip in self._skip[test.name]):
-          example.source = 'pass'
-          example.want = ''
     for example in test.examples:
-      if example.exc_msg is None:
+      if any(should_skip(example)
+             for should_skip in self._skip.get(test.name, [])):
+        example.source = 'pass'
+        example.want = ''
+        self.skipped += 1
+      elif example.exc_msg is None and any(
+          wont_implement(example)
+          for wont_implement in self._wont_implement.get(test.name, [])):
         # Don't fail doctests that raise this error.
         example.exc_msg = (
             'apache_beam.dataframe.frame_base.WontImplementError: ...')
+        self.wont_implement += 1
```

Comment:
It'd be nice to track how many actually triggered this error (and perhaps track the reasons why).

Reply:
Yeah that would be really good. Is it possible to get at that information though?

Reply:
We could record it here: https://github.com/apache/beam/blob/master/sdks/python/apache_beam/dataframe/doctests.py#L242

Reply:
I found that I could override report_success to collect this